pkg/amber/endorsement.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package amber
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+)
+
+// AmberEndorsementV2 is the ClaimType for Amber Endorsements V2. This is
+// expected to be used together with the AmberClaimV1 as the predicate type in
```

Comment:
Just checking: here "predicate type" means the same thing as "ClaimType", right?